### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by aceszhenwei**
This PR aims to update the NFR and use case (for editClient) in the DG. This commit adds the NFR.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -549,13 +549,20 @@ Use case ends.
...
 2. Should be able to hold up to 1000 Client(s) without a noticeable sluggishness in performance for typical usage.
 3. Should be able to hold up to 10 Procedure in each Client without a noticeable sluggishness in performance for typical usage.
-4. A User with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-5. The system should respond within two seconds.
 
-*{More to be added}*
+#### Quality Requirements
+1. A User with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2. Networkers should be in English.
+3. Networkers should be able to be used by colour-blind users.
```
Comment (teominghuang) [inline docs/DeveloperGuide.md:565]: Would 'Networkers is colour-blind friendly' be much better? :)

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -549,13 +549,20 @@ Use case ends.
...
+1. Networkers should respond within two seconds.
 2. Should be able to hold up to 1000 Client(s) without a noticeable sluggishness in performance for typical usage.
 3. Should be able to hold up to 10 Procedure in each Client without a noticeable sluggishness in performance for typical usage.
-4. A User with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-5. The system should respond within two seconds.
 
-*{More to be added}*
+#### Quality Requirements
+1. A User with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2. Networkers should be in English.
```
Comment (teominghuang) [inline docs/DeveloperGuide.md:564]: Would 'Networkers is currently only available in English.' be better? :)

**Review by teominghuang — CHANGES_REQUESTED** [review]
Hi, thank you for the great effort, just some suggestions to tidy up the rest of the code. :)

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (aceszhenwei) [reply docs/DeveloperGuide.md]: Good idea! Have reflected this change in the latest commit.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (aceszhenwei) [reply docs/DeveloperGuide.md]: Thanks for the review! As discussed, it has been edited to a slight variation of this suggestion.

**Review by robinrojh — APPROVED** [review]
LGTM! Good work Alexander!

**Pull request merged**